terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,63 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                 = "search-service" # this must match the stack name the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "alphabetical-company-search-consumer"
+  container_port             = 8080
+  docker_repo                = "alphabetical-company-search-consumer"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 21
+  lb_listener_paths          = ["/verification/*"]
```

Comment:
* I suspect `lb_listener_paths` may be empty in this case, as the consumer's only REST handling will be for the health check?
* For the health check, we may want to add `healthcheck_path` and `healthcheck_matcher` variables?